include/user.php

```diff
@@ -0,0 +1,128 @@
+<?php 
+
+function getUser($accountId){
+    $user = dbQuery('
+    SELECT *
+    FROM trackerSignUp
+    WHERE accountId = :accountId', 
+    [
+        'accountId' => $accountId, 
+        
+    ])->fetch();                             
+return @$user; //this is an array
+}
+
+
+
+function attemptLogin($email, $password){ 
+    $result = dbQuery('
+        SELECT accountId
+        FROM trackerSignUp
+        WHERE email = :email AND password = :password', //just a string parameter
+        [
+            'email' => $email,
+            'password' => $password
+            
+        ])->fetch();                            //where email/password matches -case insensitive email
+    return $result['accountId'];
+}    
+
+
+
+function insertNewProfileEdit($email, $age, $gender, $occupation, $occupationStressLevel, $typeBeverage, $beverageOther, $lifeEvent){
```

Comment:
Once again, this doesn't need to go in this PR, but it's a bit weird that you're taking the email parameter here instead of their accountId